REPOSITORIES.md

```diff
@@ -0,0 +1,31 @@
+## Repositories
+
+This document describes the lifecycle of repositories under the [Notary Project](https://github.com/notaryproject/) organization.
+
+Anyone can submit proposals regarding repositories' lifecycle by opening a GitHub issue in the
+[notaryproject/.github](https://github.com/notaryproject/.github) repository. The [Notary Project governance maintainers](MAINTAINERS) will take into account the community feedback and decide on the proposal.
+
+### Create a new sub-project
+
+A new sub-project can be created as a repository and contributed to the Notary Project organization by opening a GitHub issue to request [Notary Project governance maintainers](MAINTAINERS) to vote in the
+[notaryproject/.github](https://github.com/notaryproject/.github) repository. A new sub-project can be created after having a supermajority of approval from Notary Project governance maintainers.
```

Comment:
Is it a two-thirds supermajority or 60% supermajority? See https://en.wikipedia.org/wiki/Supermajority for common supermajorities. I would suggest two-thirds supermajority.

Reply:
two-thirds is fair. I clarified it